inst/tutorials/chapter3/chapter3.Rmd

```diff
@@ -145,32 +173,156 @@ y <- rlnorm(10000,meanlog=3.5,sdlog=1)
 ```
 
 ```{r, echo=FALSE}
-sliderInput("bins", "Number of bins M:", min = 1, max = 500, step=5, value = 4)
+sliderInput("binz", "Number of bins M:", min = 1, max = 500, step=5, value = 4)
 plotOutput("distPlot")
 ```
 
 ```{r, context="server"}
 output$distPlot <- renderPlot({
   y2 <- y[y<500]
-  bins <- seq(min(y), max(y), length.out = input$bins + 1)
+  bins <- seq(min(y), max(y), length.out = input$binz + 1)
   hist(y, breaks = bins, col = 'darkgray', border = 'white',freq=FALSE)
   lines(density(y),col="red",lw=2)
 })
 ```
 
+Below are some famous distributions for you to play with:
+
+| Distribution | Range | PDF $P(y)$|
+|:------------:|:-----:|:------:|
+| Normal | $(-\infty, \, +\infty)$ | $\frac{1}{{\sigma \sqrt {2\pi}}}e^{\frac{-(y-\mu)^2}{2 \sigma^2}}$|
+| LogNormal | $]0, \, +\infty)$ | $\frac{1}{y\sigma{\sqrt {2\pi}}}e^{\frac{-(\ln{y}-\mu)^2}{2 \sigma^2}}$|
+| Uniform | $[a, \, b]$ | $\frac{1}{b - a}$|
+
+Along with some more exotic distribution.
+
+```{r, echo=FALSE}
+
+shinyUI(fluidPage(
+  sidebarPanel(selectInput(inputId = "dist", # Dropdown Menu to choose a distribution
+                label = "Distribution:",
+                choices = c(Normal      = "rnorm",
+                            Lognormal   = "rlnorm",
+                            Uniform     = "runif",
+                            Beta        = "rbeta",
+                            Pareto      = "rparet",
+                            Arcsine     = "rarcsine",
+                            Ratio2Normals = "rationorm"),
+                selected = "rnorm"),
+               
+               # Change interactive parameters depending on the chosen distribution
+               
+               conditionalPanel(condition = "input.dist == 'runif'",
+                                sliderInput("range_unif", "Range", min = -100, max = 100, value = c(-5, 5))),
+               
+               conditionalPanel(condition = "input.dist == 'rbeta'",
```

Comment:
* increase ranges; alpha \in [0,3], \beta \in [0,5]
* starting values: alpha=2, beta=5